app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class UsersController < Api::V1::ApplicationController
+      def index
+        render json: UserSerializer.new(current_user)
```

Comment:
In index action you need to return all users without current user. This logic should be present in custom method, for example `current_user_details`

Reply:
Changed index action to return serialized all users without current user:
`users = User.where.not(id: current_user.id)
render json: UserSerializer.new(users)`